src/api/environment.cc

```diff
@@ -606,7 +611,7 @@ bool InitializePrimordials(Local<Context> context) {
   if (!primordials->SetPrototype(context, Null(isolate)).FromJust() ||
       !GetPerContextExports(context).ToLocal(&exports) ||
       !exports->Set(context, primordials_string, primordials).FromJust()) {
-    return false;
+    return MaybeLocal<Object>();
```

Comment:
I can’t make a multi-line suggestion here, but you probably also want to change `!….FromJust()` to `….IsNothing()` in the conditional here; because `.FromJust()` already crashes when it encounters an empty Maybe

Reply:
@addaleax done!